ArchUnitNET/Domain/MethodMember.cs

```diff
@@ -16,7 +16,7 @@ public class MethodMember : IMember
 
         public MethodMember(string name, string fullName, IType declaringType, Visibility visibility,
             ITypeInstance<IType> returnType, bool isVirtual, MethodForm methodForm, bool isGeneric, bool isStub,
-            bool isCompilerGenerated)
+            bool isCompilerGenerated, bool? isIterator)
```

Comment:
what happens if this is null?

Reply:
right now this just stores the information whether the methodmember is an iterator or nor. It can technically be used in a rule